nilearn/conftest.py

```diff
@@ -484,6 +484,33 @@ def surf_label_img(surf_mesh):
     return SurfaceImage(surf_mesh(), data)
 
 
+@pytest.fixture
+def surf_maps_img(surf_mesh):
+    """Return a sample surface map image using the sample mesh.
+    Has 6 regions in total: 3 in both, 1 only in left and 2 only in right.
+    """
+    data = {
+        "left": np.asarray(
+            [
+                [1, 1, 0, 1, 0, 0],
+                [0, 1, 1, 1, 0, 0],
+                [1, 0, 1, 1, 0, 0],
+                [1, 1, 1, 0, 0, 0],
+            ]
+        ),
+        "right": np.asarray(
+            [
+                [1, 0, 0, 0, 1, 1],
+                [1, 1, 0, 0, 1, 1],
+                [0, 1, 1, 0, 1, 1],
+                [1, 1, 1, 0, 0, 1],
+                [0, 0, 1, 0, 0, 1],
```

Comment:
given that surface maps can also be 'probability' maps should with have value between 0 and 1 here to make this fixture more general?